internal/pkg/deploy/cloudformation/cloudformation.go

```diff
@@ -219,11 +241,14 @@ func (cf CloudFormation) newRenderWorkloadInput(w progress.FileWriter, stack *cl
 	return in
 }
 
-func (cf CloudFormation) renderStackChanges(in *renderStackChangesInput) error {
+func (cf CloudFormation) executeAndRenderChangeSet(in *executeAndRenderChangeSetInput) error {
```

Comment:
```suggestion
func (cf CloudFormation) createAndRenderChangeSet(in *executeAndRenderChangeSetInput) error {
```

Is it okay if we keep the name of this method as `createAndRenderChangeSet` as `execute` sounds more of an execution of something and not creation of the stack. `createAndRenderChangeSet` name is also consistent with createAndRenderEnvironment. Not a major one so feel free to ignore if doesn't feel relevant.

Reply:
>Is it okay if we keep the name of this method as createAndRenderChangeSet as execute sounds more of an execution of something and not creation of the stack.

That's exactly right! we're executing the change set in this method that's why I renamed it. The `createChangeSet` function is a bit of misnomer I'll update in the follow-up PR, it actually either:
- creates and executes a 'Create' changeset
- or creates and executes an 'Update' changeset